docs/source/developer_zone/changes-2.0.rst

```diff
@@ -55,11 +55,23 @@ The Python bindings to the C++ ``libmamba`` library remain available through ``i
 They are now considered the first class citizen to using Mamba in Python.
 Changes include:
 
-- The global ``Context``, previously available through ``Context()``, must now be accessed through
-  ``Context.instance()``.
-  What's more, it is required to be passed explicitly in a few more functions.
-  Future version of ``libmambapy`` will continue in this direction until there are no global context.
+- The global ``Context``, previously available through ``Context()``, must now be instantiated at least
+  once before using ``libmambapy`` functions and types. Only one instance can exist at any time,
+  but that instance can be destroyed and another recreated later, for example if you use it in
+  specific functions scopes.
```

Comment:
In this case, maybe add `mamba_context = libmambapy.Context()` [here](https://github.com/mamba-org/mamba/blob/main/docs/source/usage/specs.rst?plain=1#L30) and similar lines as well?

Reply:
I'm not completely sure but I guess it's safe to assume as long as we import `libmambapy` that should be a requirement

Reply:
>  Only one instance can exist at any time

Do we have a mean to guarantee that?